contrib/busybox/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Source: http://frippery.org/busybox/
+# This Dockerfile builds a (32-bit) busybox images which is suitable for
+# running many of the integration-cli tests for Docker against a Windows
+# daemon. It will not run on nanoserver as that is 64-bit only.
+#
+# Based on http://github.com/jhowardmsft/busybox
+# John Howard (IRC jhowardmsft, Email john.howard@microsoft.com)
+#
+# To build: docker build -t busybox .
+# To publish: Needs someone with publishing rights
+ARG WINDOWS_BASE_IMAGE=mcr.microsoft.com/windows/servercore
+ARG WINDOWS_BASE_IMAGE_TAG=ltsc2019
+
+FROM ${WINDOWS_BASE_IMAGE}:${WINDOWS_BASE_IMAGE_TAG}
+RUN mkdir C:\tmp && mkdir C:\bin
+ADD http://frippery.org/files/busybox/busybox.exe /bin/
```

Comment:
Installing some random exe from an http site does not seem  acceptable. `busybox` appears to be available from `choco` can we use that?

Reply:
This URL is the "official" one to get the Windows port of busybox.
The Chocolatey package `busybox` also uses it https://github.com/chtof/chocolatey-packages/blob/master/automatic/busybox/update.ps1#L13

But, we could use https.

Reply:
Using https would be better, but if we wanted to guarantee that we're locked to a specific exe that doesn't change under our feet, perhaps grab the specific version, e.g. https://frippery.org/files/busybox/busybox-w32-FRP-3244-g48128b9aa.exe (if you really need to get the 32-bit ver) and `sha256sum` check it to hard-coded `068744fbe3308807c7658da711821af13af6563dab6d86f7aca1ac45e3710eba`